From a525c66e6b221e6249949dc3d52d4f981a6e32c7 Mon Sep 17 00:00:00 2001
From: Michel JAOUEN <michel.jaouen@st.com>
Date: Thu, 5 Sep 2024 16:46:37 +0200
Subject: [PATCH 096/120] misc: stm32-ipcc: fix binding issue

The proc id is locate at index 0

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/u-boot.git eef4e2b0ec1d436d1b82d9071c8ad97e566951df misc: stm32-ipcc: fix binding issue]
Change-Id: I14c657f20af309bc38a142ac41e95b6a632dbd7c
Signed-off-by: Michel JAOUEN <michel.jaouen@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/u-boot/+/403143
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 drivers/mailbox/stm32-ipcc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mailbox/stm32-ipcc.c b/drivers/mailbox/stm32-ipcc.c
index ff1e439c54f..a6455e8d647 100644
--- a/drivers/mailbox/stm32-ipcc.c
+++ b/drivers/mailbox/stm32-ipcc.c
@@ -114,7 +114,7 @@ static int stm32_ipcc_probe(struct udevice *dev)
 	ipcc->reg_base = (void __iomem *)addr;
 
 	/* proc_id */
-	ret = dev_read_u32_index(dev, "st,proc_id", 1, &ipcc->proc_id);
+	ret = dev_read_u32_index(dev, "st,proc-id", 0, &ipcc->proc_id);
 	if (ret) {
 		dev_dbg(dev, "Missing st,proc_id\n");
 		return -EINVAL;
-- 
2.47.2

